### PR TITLE
first add beniget conda-forge recipe

### PR DESCRIPTION
**Pull request by CyrilleBonamy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by paugier** [comment]
Thanks Cyrille. I confirm.

File: recipes/beniget/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - gast >=0.2.2
+    - python
+
+test:
+  imports:
+    - beniget
+
+about:
+  home: https://github.com/serge-sans-paille/beniget/
+  license: BSD3
```
Comment (chrisburr) [inline recipes/beniget/meta.yaml]: From https://opensource.org/licenses/alphabetical:
```suggestion
  license: BSD-3-Clause
```

**Comment by chrisburr** [comment]
I'm not sure what went wrong with the CI, let's try again...
@conda-forge-admin, please lint

**Comment by CyrilleBonamy** [comment]
> I'm not sure what went wrong with the CI, let's try again...
> @conda-forge-admin, please lint

thanks a lot  @chrisburr

**Pull request merged**